Source/WebCore/css/StyleSheetContents.h

```diff
@@ -45,7 +46,7 @@ class StyleRuleNamespace;
 
 enum class CachePolicy : uint8_t;
 
-class StyleSheetContents final : public RefCounted<StyleSheetContents>, public CanMakeWeakPtr<StyleSheetContents> {
+class StyleSheetContents final : public RefCounted<StyleSheetContents>, /*public CanMakeWeakPtr<StyleSheetContents>,*/ public CanMakeCheckedPtr {
```

Comment:
Please don’t forget to delete the commented-out part here. Maybe also delete the include of `WeakPtr.h`?

Reply:
Oh wow. Sorry about that. I really miss the pretty-diff that `webkit-patch` was showing before uploading the patch. I would always catch mistakes like these thanks to it.